samples/map/arithmetic_add/README.md

```diff
@@ -0,0 +1,9 @@
+**arithmetic_add**
+
```

Comment:
Rewrite as follows:

This example implements the computation of the addition of the first *n* natural numbers.

The program first creates a vector with the n first natural numbers. A map pattern is used to ...